tools/p4dbg.py

```diff
@@ -123,8 +133,8 @@ def get_all_fields():
     return field_map.get_all_fields()
 
 def enum(type_name, *sequential, **named):
```

Comment:
we should consider dropping support for Python2 altogether in a future PR, then we can use `enum.Enum`